bodhi/tests/server/functional/test_comments.py

```diff
@@ -13,9 +13,12 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 from datetime import datetime, timedelta
-
 import mock
+import copy
```

Comment:
```copy``` is a built-in, and technically for Python 2 ```mock``` is still third-party. So let's group mock with ```webtest``` and ```copy``` with ```datetime```.